resources/lang/en-US/validation.php

```diff
@@ -187,6 +187,8 @@
 
     'custom' => [
         'alpha_space' => 'The :attribute field contains a character that is not allowed.',
+        'alpha_encrypted'   => 'The :attribute field should be alphabetic characters.',
```

Comment:
We should be able to use the existing validation strings for this, no?

Reply:
Derp, yeah, resolved.